docs/docs/03-light-clients/04-wasm/03-integration.md

```diff
@@ -155,11 +155,11 @@ import (
 
 // instantiate the Wasm VM with the chosen parameters
 wasmer, err := wasmvm.NewVM(
-  dataDir, 
-  availableCapabilities, 
-  contractMemoryLimit,
-  contractDebugMode, 
-  memoryCacheSize,
+  dataDir,
+  availableCapabilities,
+  contractMemoryLimit, // default of 32
```

Comment:
wasmvm does not expose this in any way, just a nice to have to let others know the default